lib/crypt_keeper/model.rb

```diff
@@ -119,6 +119,9 @@ def decrypt_table!
       # Private: The encryptor class
       def encryptor_klass
         @encryptor_klass ||= "CryptKeeper::Provider::#{crypt_keeper_encryptor.to_s.camelize}".constantize
+      rescue NameError
+        # couldn’t constantize... 
+        nil
       end
```

Comment:
@nikz is this something you ran into while testing and playing around? Do you think a test case is needed?

Reply:
hiya @jmazzi, it seems like the behaviour for [`constantize` has changed in later versions of Rails/Ruby](https://github.com/rails/rails/blob/main/activesupport/lib/active_support/inflector/methods.rb#L276), and it now raises a `NameError` instead of returning nil. Happy to put a test case in there, but I felt like it was probably covered in ActiveSupport? 

Looks like the change was introduced in [this commit](https://github.com/rails/rails/commit/7057ccf6565c1cb5354c1906880119276a9d15c0)?